flask_resty/pagination.py

```diff
@@ -483,7 +483,7 @@ def decode_value(self, value: str):
         value += (3 - ((len(value) + 3) % 4)) * b"="  # Add back padding.
         value = base64.urlsafe_b64decode(value).decode()
 
-        return None if value == "None" else value
+        return None if value == str(sa.null()) else value
```

Comment:
nitpick (non-blocking): can probably just use `"NULL"` here or make a constant.

```python
_NULL = str(sa.null())

def decode_value(..):
   return ... if value == _NULL else value
   
def encode_value(..):
  if value is None:
    value = _NULL
  else:
    value = str(value)
```

this saves a bit of type-casting (which normally wouldn't matter but this is a rather "hot" code path, as the encoding/decoding happens for every item in the queryset)

Reply:
done